cob_generic_can/CMakeLists.txt

```diff
@@ -1,27 +1,30 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(cob_generic_can)
 
-find_package(catkin REQUIRED COMPONENTS cob_utilities libntcan libpcan)
+find_package(catkin REQUIRED COMPONENTS socketcan_interface cob_utilities libntcan libpcan)
```

Comment:
please respect alphabetical order throughout this file (find_package, CATKIN_DEPENDS,...)